app/controllers/provider_interface/provider_users_invitations_controller.rb

```diff
@@ -98,6 +98,19 @@ def previous_page
 
   private
 
+    def available_providers
+      current_provider_user.authorisation.providers_that_actor_can_manage_users_for
+    end
+
+    def wizard_setup_options
+      setup_options = { current_step: 'details' }
+      if available_providers.count == 1
+        setup_options[:providers] = available_providers.map(&:id)
+        setup_options[:single_provider] = true
```

Comment:
Of course, it makes sense to pass this in so the wizard doesn't know about current_provider_user